test/parallel/test-sync-fileread.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
 
-var fixture = path.join(common.fixturesDir, 'x.txt');
+const fixture = path.join(common.fixturesDir, 'x.txt');
 
-assert.equal('xyz\n', fs.readFileSync(fixture));
+assert.strictEqual('xyz\n', fs.readFileSync(fixture, 'utf8'));
```

Comment:
Why the addition of `'utf8'`? We should still test the case where the encoding is not provided. If anything, maybe add another assertion.